lib/ariadne/data_util.rb

```diff
@@ -1,11 +1,12 @@
 
 module DataUtil
-  def self.init_redis_cli(obj = nil)
-    @redis_cli = obj
+  def self.init_redis_cli(redis_obj: nil)
+    @redis_cli = redis_obj
```

Comment:
should this not be in a begin-rescue-end block? given that `redis_obj` can be `nil`